Sources/HTTP/Responder/HTTPClient.swift

```diff
@@ -193,3 +197,16 @@ private enum HTTPClientState {
     /// Currently parsing the response's body.
     case parsingBody(HTTPResponseHead, Data?)
 }
+
+extension String {
+    func isIPAddress() -> Bool {
```

Comment:
This should go in a separate file. Either a `Utilities.swift` file (which might already exist) or something like `String+IsIPAddress.swift`.